app/src/main/res/values/strings.xml

```diff
@@ -171,6 +171,8 @@
     <string name="you_just_need_choose_the_compensation_from_time">You just need choose the compensation from time, the to time will auto </string>
     <string name="compensation_time_is_not_in_the_past">Compensation time is not in the past</string>
     <string name="your_compensation_time_can_not_in_working_time">Your compensation time can not in working time of company</string>
+    <string name="check_in_time">Checkin time</string>
```

Comment:
FATAL: [MissingTranslation] "`check_in_time`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)